hydra/conf/hydra/hydra_logging/notset.yaml

```diff
@@ -0,0 +1,5 @@
+# @package _group_
+
+version: 1
+root: ~
```

Comment:
Why `~` and not `null`?

Reply:
I didn't even realize `null` was used for None, TIL. I've always used `~` but I guess I'm switching over to `null` now!